pkg/gather/clusterconfig/openshift_authentication_logs.go

```diff
@@ -15,7 +15,8 @@ func GatherOpenshiftAuthenticationLogs(g *Gatherer, c chan<- gatherResult) {
 	defer close(c)
 
 	containersFilter := logContainersFilter{
-		namespace: "openshift-authentication",
+		namespace:     "openshift-authentication",
+		labelSelector: "oauth-openshift",
```

Comment:
I think it should be `app=oauth-openshift`

Reply:
Ohh yeah. Good catch. Thanks